### PR TITLE
chore: Fix clippy errors on beta

### PR DESCRIPTION
**Pull request by kjarosh**

### CODE REVIEW

**Comment by sleepycatcoding** [comment]
It seems there are more occurrences of the lint.

**Comment by danielhjacobs** [comment]
Testing locally, there's a lot more then that. To test this locally:

- `rustup update`
- `rustup default beta`
- `cargo clippy --all --all-features --tests -- -D warnings`
- Make changes
- Run cargo command again

Don't forget to switch back to rustup default stable after fixing all occurences.

**Comment by kjarosh** [comment]
:sob: I will fix other occurrences in a minute. I was wondering why only one error was raised in the CI...

**Comment by torokati44** [comment]
Instead of switching the default, you can always: `cargo +beta clippy ...`

**Review by sleepycatcoding — APPROVED** [review]

**Comment by kjarosh** [comment]
The remaining issue that fails is `std::panic::PanicInfo` renamed to `std::panic::PanicHookInfo` (https://releases.rs/docs/1.81.0/#compatibility-notes), which was released in 1.81.0 and deprecated in 1.82.0. Unfortunately beta is 1.82 already but stable is still 1.80.

**Comment by torokati44** [comment]
That's supposed to change tomorrow... :)
https://www.whatrustisit.com/

File: swf/src/types.rs [hunk 1]
```diff
@@ -1786,6 +1786,7 @@ pub type DoAction<'a> = &'a [u8];
 pub type JpegTables<'a> = &'a [u8];
 
 /// `ProductInfo` contains information about the software used to generate the SWF.
```
Comment (torokati44) [inline swf/src/types.rs:1788]: nit: Repeating the type name is redundant here, just like it was for `enum_trait_object`.

File: render/src/pixel_bender.rs [hunk 2]
```diff
@@ -272,7 +272,9 @@ pub enum PixelBenderShaderArgument<'a> {
     },
 }
 
-/// An image input. This accepts both an owned BitmapHandle,
+/// An image input.
+///
+/// This accepts both an owned BitmapHandle,
 /// and a borrowed texture (used when applying a filter to
 /// a texture that we don't have ownership of, and therefore
 /// cannot construct a BitmapHandle for).
```
Comment (torokati44) [inline render/src/pixel_bender.rs:280]: nit: I probably would have re-wrapped this neatly into 3 lines, but oh well...

**Review by torokati44 — APPROVED** [review]
I think the `PanicInfo` rename can be dealt with in a separate PR later. Thank you!

File: swf/src/types.rs (hunk 1 above)
Comment (kjarosh) [reply swf/src/types.rs]: fixed

File: render/src/pixel_bender.rs (hunk 2 above)
Comment (kjarosh) [reply render/src/pixel_bender.rs]: fixed

**Pull request merged**